src/components/CourseCard.astro

```diff
@@ -21,16 +21,30 @@ const { course, holiday = false, prices } = Astro.props;
   <div
     class="mb-0 shrink-0 overflow-hidden md:mb-0 md:min-h-full md:rounded-l-2xl"
   >
-    <a href={`/courses/${course.slug}`}>
-      <div class="h-72 w-full overflow-hidden md:h-full md:w-64 lg:w-96">
+    <div class="relative h-72 w-full overflow-hidden md:h-full md:w-64 lg:w-96">
+      <a href="href={`/courses/${course.slug}`}">
         <Picture
           src={course.data.heroImage}
           alt={`${course.data.title}`}
           class="min-h-full min-w-full object-cover object-center"
           formats={["avif", "webp"]}
         />
+      </a>
+      <div class="absolute bottom-0 flex w-full">
```

Comment:
This course card component is used on the frontpage of the site so anything specific to the holiday card should be behind conditional checks as to not affect non-holiday content.

Reply:
Correct. Added checks and moved the links to their own component